SpellChecker.java

```diff
@@ -8,28 +8,70 @@ public static void main(String[] args) {
 		String[] dictionary = readDictionary("dictionary.txt");
 		String correction = spellChecker(word, threshold, dictionary);
 		System.out.println(correction);
+
 	}
 
 	public static String tail(String str) {
-		// Your code goes here
+		String tail = "";
+		if (str.length() == 1){
+			 return tail;
+		}
+		tail = str.substring(1);
```

Comment:
You can reduce multiply `return` statements that return same variable if you change the condition to be as follow:
```suggestion
		if (str.length() > 1){
			tail = str.substring(1);
		}
```